airbyte-integrations/connectors/source-postgres/src/main/resources/spec.json

```diff
@@ -254,16 +240,14 @@
               "method": {
                 "type": "string",
                 "const": "CDC",
-                "enum": ["CDC"],
-                "default": "CDC",
                 "order": 0
               },
               "plugin": {
                 "type": "string",
                 "title": "Plugin",
                 "description": "A logical decoding plugin installed on the PostgreSQL server. The `pgoutput` plugin is used by default. If the replication table contains a lot of big jsonb values it is recommended to use `wal2json` plugin. Read more about <a href=\"https://docs.airbyte.com/integrations/sources/postgres#step-2-select-a-replication-plugin\">selecting replication plugins</a>.",
                 "enum": ["pgoutput", "wal2json"],
-                "default": "pgoutput",
+                "const": "pgoutput",
```

Comment:
was this change intended? It seems to have removed the ability for users to switch to a different plugin https://github.com/airbytehq/oncall/issues/1133

Reply:
submitted https://github.com/airbytehq/airbyte/pull/19985, if this passes SATs then I assume things are fine :shrug:

Reply:
The SAT forced me to use `const` only.